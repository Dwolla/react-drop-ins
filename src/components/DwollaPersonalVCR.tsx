import type { FC } from 'react';
import { PersonalVCRAttributes } from '../@types/component-attributes';

const DwollaPersonalVCR: FC<PersonalVCRAttributes<any>> = (props) => {
  return <dwolla-customer-create {...props} />;
};

export default DwollaPersonalVCR;
