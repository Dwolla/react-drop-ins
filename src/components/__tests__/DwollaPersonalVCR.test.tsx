import TestRenderer from 'react-test-renderer';
import DwollaPersonalVCR from '../DwollaPersonalVCR';

test('DwollaPersonalVCR', () => {
  const tree = TestRenderer.create(<DwollaPersonalVCR />).toJSON();

  expect(tree).toMatchSnapshot();
});

test('with optional props', () => {
  const tree = TestRenderer.create(
    <DwollaPersonalVCR
      correlationId="123abc"
      email="janedoe@email.com"
      firstName="Jane"
      lastName="Doe"
      ipAddress="255.255.255.255"
      privacy="www.yourprivacy.com"
      terms="www.yourterms.com"
      customerId="123"
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
