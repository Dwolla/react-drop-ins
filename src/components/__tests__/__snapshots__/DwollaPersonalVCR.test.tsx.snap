// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaPersonalVCR 1`] = `<dwolla-customer-create />`;

exports[`with optional props 1`] = `
<dwolla-customer-create
  correlationId="123abc"
  customerId="123"
  email="janedoe@email.com"
  firstName="Jane"
  ipAddress="255.255.255.255"
  lastName="Doe"
  privacy="www.yourprivacy.com"
  terms="www.yourterms.com"
/>
`;
